whoami/whoami.go

```diff
@@ -73,10 +72,9 @@ func main() {
 		os.Exit(0)
 	}
 
-	uid := strconv.Itoa(os.Geteuid())
-	u, err := user.LookupId(uid)
+	u, err := user.Current()
 	if err != nil {
-		fatal.Fatalf("cannot find name for user ID %s\n", uid)
+		fatal.Fatalln("cannot find name for current user")
```

Comment:
I'm not a huge fan of having the outputs be different... but I couldn't find a way around it.